quarkus-workshop-super-heroes/docs/src/docs/asciidoc/core-introduction/introduction-presentation.adoc

```diff
@@ -153,18 +183,17 @@ ifdef::use-native[]
 * GraalVM {graalvm-version}
 endif::use-native[]
 * Docker or Podman
-* cURL (or any other command line HTTP client)
+* cURL (or any other command line HTTP client) and optionally `jq` (check the xref:appendix-installing-jq[appendix for installation instructions])
```

Comment:
I don't think jq is optional, is it? The instructions refer to it, so it's necessary. Well, it's only used in the azure and docker sections, so we could do a guard for those, to keep the prereqs shorter. That will be a problem if another section starts using jq, but that might be an ok risk to take.